### PR TITLE
Testing comments positioning (Maxim)

### PR DESCRIPTION
**Pull request by maxim5**
comment

### CODE REVIEW

File: my.test [hunk 1]
```diff
@@ -1,11 +1,10 @@
 as
 dfsg
```
Comment (maxim5) [inline my.test]: [up] sadf111

File: my.test [hunk 2]
```diff
@@ -1,11 +1,10 @@
...
 dsaf
-sadf
-sdfa
-web
-dsfg
-sad
```
Comment (maxim5) [inline my.test]: [up] web33441111

File: my.test [hunk 3]
```diff
@@ -1,11 +1,10 @@
...
 
-123sadf
+sadf111
+sdfa
```
Comment (maxim5) [inline my.test]: [up] 123sadf

File: my.test (hunk 2 above)
Comment (maxim5) [inline my.test]: web33441111

File: 123 [hunk 4]
```diff
@@ -3,8 +3,9 @@ qwe
 
 ++
 000
-
+sadf
```
Comment (iluxa-com) [inline 123:6]: 3.5 inline test

**Comment by iluxa-com** [comment]
sadfa

**Pull request closed without merging**